addons/gdcef/gdcef/src/gdbrowser.hpp

```diff
@@ -131,6 +132,11 @@ class GDBrowserView : public godot::Node
             return this;
         }
 
+        virtual CefRefPtr<CefAudioHandler> GetAudioHandler() override
+        {
+            return this;
```

Comment:
This code will force using your audio callback. I do not like that since your code is still experimental and this force to add a AudioStreamPlayer3D node in all projects.  I would `return this` if and only if `godot::Dictionary browser_settings` has a field `{"audio_rerouting", STATE_ENABLED}` else `return nullptr`to use the default sound pipeline. 

See: https://github.com/Lecrapouille/gdcef/blob/4b0fac54e4213c14409e33588d4f516d651b0b81/addons/gdcef/gdcef/src/gdcef.hpp#L243-L264